app/payment/channel.go

```diff
@@ -83,3 +97,26 @@ func GetBalInfo(ch perun.ChannelAPI) perun.BalInfo {
 	chInfo := ch.GetInfo()
 	return balsFromState(chInfo.Currency, chInfo.State, chInfo.Parts)
 }
+
+// SubPayChUpdates registers a subscription for updates on this channel.
```

Comment:
```suggestion
// SubPayChUpdates subscribe for updates on this channel.
```

Reply:
Changed to sets up a subscription for ...